Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -78,195 +81,234 @@ Copyright Notice
+   provide sufficient insight into the operations of the system as a
+   whole that it is straightforward to imagine structural changes to the
+   root server system's infrastructure, and to wonder what the
+   operational implications of such changes might be.
+
+   The Yeti DNS Project was conceived in May 2015 to provide a captive,
+   non-production testbed upon which the technical community could
+   propose and run experiments designed to answer these kinds of
+   questions.  Coordination for the project was provided by TSIF, the
+   WIDE Project and the Beijing Internet Institute.  Many volunteers
+   collaborated to build a distributed testbed that at the time of
+   writing includes 25 Yeti root servers with 15 operators and handles
+
+
+
+Song, et al.            Expires December 18, 2017               [Page 3]
+
+Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
+
+
+   experimental traffic from individual volunteers, universities, DNS
+   vendors and distributed measurement networks.
 
-   To benefit Internet development as a whole, the Yeti DNS Project was
-   proposed to build a parallel, experimental, live IPv6 DNS root system
-   to discover the limits of DNS root name service and deliver useful
-   technical output.  Possible research agenda will be explored on this
-   testbed, covering several aspects (but not limited to):
+   By design, the Yeti testbed system serves the root zone published by
+   the IANA with only those structural modifications necessary to ensure
+   that it is able to function usefully in the Yeti testbed system
+   instead of the production Root Server system.  In particular, no
+   delegation for any top-level zone is changed, added or removed from
+   the IANA-published root zone to construct the root zone served by the
+   Yeti testbed system.  In this document, for clarity, we refer to the
+   zone derived from the IANA-published root zone as the Yeti-Root zone.
 
-   o  IPv6-only operation
+   This document presents a summary of the experiments carried out as
+   part of the Yeti DNS Project during its first two years of operation.
 
-   o  DNSSEC key rollover
+2.  Areas of Study
 
-   o  Renumbering issues
+   The experiments described in this document are grouped loosely into
+   multiple sets, each representing a somewhat distinct set of problem
+   statements.  These areas of study are summarised here in the form of
+   lists of questions to give context to the experiments that follow.
 
-   o  Scalability issues
+   The infrastructure on which the experiments have been run has also
+   involved significant effort to build, principally in its operational
+   aspects: the coordination of a large pool of volunteers in the
+   implementation of the testbed is itself an exercise with relevance to
+   the real world, and commentary on the corresponding logistics is also
+   included in this document.
 
-   o  Multiple zone file signers
+2.1.  Yeti-Root Zone Distribution
```

Comment:
Is it worthwhile to introdce a very brief context of Root zone distribution. like Anycast as mentioned in the introduction, RFC7706 which is well awared by audiences, and also unowned  anycast proposaed by Paul, and the alternative root servers is another way. That context may brief the audiences the backgroud on this issue. So that they can understand why Yeti study on this.